os/kernel/init/os_bringup.c

```diff
@@ -284,6 +256,17 @@ static inline void os_do_appstart(void)
 
 	svdbg("Starting application init thread\n");
 
+#ifdef CONFIG_BINARY_MANAGER
+#define BINARY_MANAGER_STACKSIZE 2048
+#define BINARY_MANAGER_PRIORITY 202
```

Comment:
Is there any specific reason to choose the priority as 202. What if the tasks have higher priority than this.